scripts/verify_verification_trigger_contract.py

```
@@ -0,0 +1,319 @@
+    lines = ci_text.splitlines()
+    filters_start = None
+    filters_indent = None
+
+    for idx, line in enumerate(lines):
+        if line.strip() == "filters: |":
+            filters_start = idx + 1
+            filters_indent = len(line) - len(line.lstrip(" "))
+            break
+
+    if filters_start is None or filters_indent is None:
+        raise ValueError("ci.yml missing dorny/paths-filter literal filters block")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)